src/main/java/teamrtg/rtg/core/RTG.java

```diff
@@ -38,11 +44,12 @@ public void preInit(FMLPreInitializationEvent event) {
 
         configPath = event.getModConfigurationDirectory() + "/RTG/";
         Mods.syncAllConfigs();
+        
+        // This must come before the event manager is registered.
+        this.registerStructures();
```

Comment:
I guess I was just trying to keep the preInit a bit cleaner, but happy to move them back.

They might not *need* to come before the event manager is registered, but I think they *should* because InitMapGen references them, and in general the sooner they're registered the better in case any other mods depend on them being available early.